tldw_Server_API/app/core/AuthNZ/user_provider_secrets.py

```
@@ -14,6 +14,22 @@ def normalize_provider_name(provider: str) -> str:
     return (provider or "").strip().lower()
 
 
+def normalize_secret_owner_scope_type(scope_type: str) -> str:
+    normalized = (scope_type or "").strip().lower()
+    if normalized in {"user", "users"}:
+        return "user"
+    if normalized in {"org", "organization", "organizations", "orgs"}:
+        return "org"
+    if normalized in {"team", "teams"}:
+        return "team"
+    raise ValueError(f"Invalid secret owner scope type: {scope_type}")
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)